notebooks/how_to/run_tests_that_require_multiple_datasets.ipynb

```diff
@@ -308,6 +389,35 @@
                 "    inputs={\"datasets\": (vm_train_ds, vm_test_ds), \"model\": vm_model_xgb},\n",
                 ")"
             ]
+        },
+        {
+            "cell_type": "markdown",
+            "metadata": {},
+            "source": [
+                "<a id='toc9_'></a>\n\n",
+    "## Next steps\n",
+                "\n",
+                "You can look at the output produced by the ValidMind Developer Framework right in the notebook where you ran the code, as you would expect. But there is a better way: use the ValidMind platform to work with your model documentation.\n",
```

Comment:
```suggestion
    "You can look at the results of this test suite right in the notebook where you ran the code, as you would expect. But there is a better way&mdash;view the prompt validation test results as part of your model documentation in the ValidMind Platform UI:\n",
```

Reply:
Changed globally to use the emdash character, with spaces before and after.